.evergreen/.evg.yml

```diff
@@ -736,7 +736,7 @@ buildvariants:
     - name: "reactive-streams-tck-test"
 
 - matrix_name: "scala-tests"
-  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "jdk11", version: ["4.2"], topology: "standalone", scala: "*", os: "ubuntu" }
+  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "jdk11", version: ["4.2"], topology: "sharded-cluster", scala: "*", os: "ubuntu" }
```

Comment:
Why sharded only? If I were to pick one I would pick replica-set (greater chance of success, most common deployment)

Reply:
Sounds good - updated.